pubnub-chat-api/src/commonMain/kotlin/com/pubnub/chat/Chat.kt

```diff
@@ -136,58 +339,166 @@ interface Chat {
         membershipCustom: CustomObject? = null,
     ): PNFuture<CreateGroupConversationResult>
 
+    /**
+     * Lets you watch a selected channel for any new custom events emitted by your chat app.
+     *
+     * @param type Reified type parameter bounded by the EventContent interface, allowing access to type information at runtime
+     * e.g. EventContent.Receipt::class, EventContent.TextMessageContent::class
+     * @param channelId Channel to listen for new events.
+     * @param customMethod An optional custom method for emitting events. If not provided, defaults to null.
+     * @param callback A lambda function that is called with an Event<T> as its parameter.
+     * It defines the custom behavior to be executed whenever an event is detected on the specified channel.
+     *
+     * @return AutoCloseable Interface you can call to stop listening for new messages and clean up resources when they
+     * are no longer needed by invoking the close() method.
+     */
     fun <T : EventContent> listenForEvents(
```

Comment:
the extension function with the same name is not documented

Reply:
Done